package/yast2-samba-server.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-samba-server
-Version:        3.1.16
+Version:        3.1.17
```

Comment:
ok thanks for the heads up!, I'll try fix up this and the other comment later today